Matchmaker/CassandraMatchRequestRepository.cs

```diff
@@ -1,47 +1,69 @@
 using Cassandra.Mapping;
+using Cassandra;
 
 public class CassandraMatchRequestRepository : IMatchRequestRepository
 {
     private IMapper db;
+    private ConsistencyLevel _consistencyLevel;
     private Random random = new();
 
-    public CassandraMatchRequestRepository(IMapper cassandraMapper)
+    public CassandraMatchRequestRepository(int port, ConsistencyLevel consistencyLvl)
     {
-        db = cassandraMapper;
+        var cluster = Cluster.Builder()
+                     .AddContactPoint("127.0.0.1")
+                     .WithPort(port)
+                     .Build();
+
+        var session = cluster.Connect("matchmaker");
+
+        db = new Mapper(session);
```

Comment:
Ten `Mapper` może być tworzony na zewnątrz, np. w `MatchmakerSimulator`, a tu wystarczy DI

Reply:
Podobnie w pozostałych repozytoriach

Reply:
To można sprawdzić, czy czasy wychodzą inne

Reply:
Zresztą teraz nawet pojedynczy matchmaker ma 3 oddzielne sesje, więc to jest zdecydowanie nadmiarowe